src/sentry/static/sentry/app/views/settings/incidentRules/triggers/chart/index.tsx

```diff
@@ -213,6 +213,7 @@ class TriggersChart extends React.PureComponent<Props, State> {
               yAxis={aggregate}
               includePrevious={false}
               currentSeriesName={aggregate}
+              partial={false}
```

Comment:
As discussed with @wedamija, we do not want to include the last partial bucket in the time series when looking at alerts.